README.md

```diff
@@ -70,4 +70,6 @@ formMapper depends on the following browser APIs:
 
 As of Febrary 2018, matches is supported in IE9 and up.
 
++Array.isArray: [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) | [Polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill)
```

Comment:
This has support back to IE9, so you don't need it.

Reply:
OUTDATED